catalog/docs/src/main/resources/_contents/extending-contents.adoc

```diff
@@ -3633,6 +3633,54 @@ Use the Tika Input Transformer for ingesting Microsoft documents, OpenOffice doc
 
 This input transformer maps the metadata common across all mime types to applicable metacard attributes in the default MetacardType.
 
+===== PPTX Input Transformer
+
+The PPTX Input Transformer is input transformer responsible for translating Microsoft PowerPoint (OOXML only) documents into a Catalog Metacard. This input transformer utilizes Apache Tika
+(for basic metadata) and Aapche POI (for thumbnail creation). The PPTX Input Transformer's main purpose is to ingest PPTX documents into the ${branding} Content Repository and the Metadata Catalog.
+
+The PPTX Input Transformer will take precedence over the Tika Input Transformer for PPTX documents.
+
+====== Installing and Uninstalling
+This transformer is installed by default. To install or uninstall manually, use the `catalog-transformer-pptx` feature in the ${admin-console} (https://localhost:8993/admin) under DDF Catalog -> Features.
+
+
+====== Configuring
+None
```

Comment:
As an end user, this sounds awkward. How about `This transformer does not require any configuring` for a more formal expression?